### ISSUE TITLE
localstorage doesn't allow '' as a key

### ISSUE THREAD

**Issue by tinybike** [lmaccherone/node-localstorage#16]
node-localstorage is working nicely for the most part:
```javascript
> localStorage.setItem(":)", "hello world");
undefined
> localStorage.getItem(":)");
'hello world'
```
However, it throws an error when I try to use `''` as the key:
```javascript
> localStorage.setItem('', 'hello world');
Error: EISDIR: illegal operation on a directory, open 'scratch'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.writeFileSync (fs.js:1161:15)
    at LocalStorage.setItem (/home/jack/src/augur.js/node_modules/node-localstorage/LocalStorage.js:155:10)
    at repl:1:14
    at REPLServer.defaultEval (repl.js:164:27)
    at bound (domain.js:250:14)
    at REPLServer.runBound [as eval] (domain.js:263:12)
    at REPLServer.<anonymous> (repl.js:393:12)
    at emitOne (events.js:82:20)
```
(I've used `''` as a localStorage key in both Firefox and Chrome and it works fine there.)

**Comment by lmaccherone** [comment]
Good catch. I'll write a test to confirm the issue and then implement a fix.

**Comment by lmaccherone** [comment]
OK. Fixed in v1.0.0. Thanks!!!
